src/juju_doctor/builtin/application-exists.py

```diff
@@ -0,0 +1,89 @@
+"""Application-exists verbatim builtin plugin."""
+
+import logging
+from typing import Dict, Optional
+
+from pydantic import BaseModel, ConfigDict, Field
+from rich.logging import RichHandler
+
+from juju_doctor.artifacts import read_file
+
+logging.basicConfig(level=logging.WARN, handlers=[RichHandler()])
+log = logging.getLogger(__name__)
+
+
+class ApplicationAssertion(BaseModel):
```

Comment:
Would we benefit from a convention where the (top level) pydantic model is named after the builtin? In this case it would be `class ApplicationExists`.